packages/ckeditor5-core/src/typings.ts

```diff
@@ -12,3 +12,7 @@ export type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
 export type NonEmptyArray<A> = Array<A> & {
 	0: A;
   };
+
+export type Maybe<A> = A | null | undefined;
```

Comment:
1. I got rid of this type in ckeditor5-ui and nothing changed. What particular problem did it solve?
2. If it was really necessary, let's keep it in ckeditor5-ui since this is the only place that needs it.

Reply:
1. It's syntax sugar. More-less implementation of `Option` from https://mobily.github.io/ts-belt/api/option/. I'd say it makes code look less verbose and at the same time more functional by adding reference to `Option` / `Maybe` Haskell monad. If it is unacceptable, then I'll remove it.  
2. I'd suggest keeping it in core because it is a core utility that is not related directly to `ui` and will be used more and more times by programmers familiar with framework(s) specified in point before.

Reply:
Removed after discussion.